From 3fb97848be8b93b5db9c8a1db2cf544cfbdc0e51 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 29 Aug 2018 15:14:25 +0800
Subject: [PATCH 34/35] arm: dts: am335x-evm: enable BT by uart1

The original commit comes from:
      git://git.ti.com/processor-sdk/processor-sdk-linux.git

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index 66aab24..7527dbf 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -140,6 +140,18 @@
 		};
 	};
 
+	kim {
+		compatible = "kim";
+		nshutdown_gpio = <117>; /* Bank3, pin21 */
+		dev_name = "/dev/ttyS1";
+		flow_cntrl = <1>;
+		baud_rate = <3000000>;
+	};
+
+	btwilink {
+		compatible = "btwilink";
+	};
+
 	sound {
 		compatible = "simple-audio-card";
 		simple-audio-card,name = "AM335x-EVM";
-- 
1.7.9.5

